_posts/2020-11-30-using-your-own-product-a-headless-cms-case-study.md

```diff
@@ -0,0 +1,53 @@
+---
+
+title: "Using your own product - a headless cms case study"
+author: [herflis]
+image: "../img/posts/dogfood.jpg"
+tags: [website, headless cms, case study, project]
+redirect_to: https://www.sensenet.com/blog/2020-11-30-using-your-own-product-a-headless-cms-case-study
+
+---
+
+Eat your own dogfood they said. It'll be fun they said...
+
+---
+
+If you followed us, you could have read it a million times already, we are about to launch sensenet in SaaS model which is a huge change from selling a traditional (E)CMS to serve our customer with a cloud-based, headless CMS. These changes not only affected our sales and development company processes but also our communication and at the same time it maked necessary to update our public website. Nothing is simpler, since the product we develop is perfect for creating public websites (among many other things). The old website was built with the previous version of the product which was a complex ECMS, so the time was perfect to not only redesign the site and update content, but for looking through a specific problem at what solution we offer to it.
```

Comment:
```suggestion
If you followed us, you could have read it a million times already, we are about to launch sensenet in SaaS model - which is a huge change from selling a traditional (E)CMS - to serve our customer with a cloud-based, headless CMS. These changes not only affected our sales and development processes but also our communication and at the same time it made necessary to update our public website. Nothing is simpler, since the product we develop is perfect for creating public websites (among many other things). The old website was built with the previous version of the product which was a complex ECMS, so the time was perfect not only to redesign the site and update every content, but to test the new model in this specific scenario.
```